app/assets/stylesheets/alchemy/selects.scss

```diff
@@ -17,7 +17,7 @@ select {
       @include button($base-color: $white, $padding: 0 5px);
       display: block;
       font-weight: normal;
-      border-color: $button-border-color;
```

Comment:
Oh, my bad. The buttons and selects should use `$button-border-color`, not `$default-border-color`, sorry.

Actually there are [lots of `button` variables](https://github.com/AlchemyCMS/alchemy_cms/blob/master/app/assets/stylesheets/alchemy/_variables.scss#L31-L36) we should use to style the buttons.